docs/access_and_permissions.md

```diff
@@ -121,6 +121,61 @@ flowchart TD
     D --Pundit.policy_scope(current_user, SupportedPermission)--> G("SupportedPermissionPolicy (scope)")
 ```
 
+#### Account signin permissions create
+
+These dependencies determine whether a user can:
+
+- complete the controller action
+- grant themself access to an app
+
+```mermaid
+flowchart TD
+    A(Account::SigninPermissionsController#create)
+    --authorize [:account, Doorkeeper::Application], :grant_signin_permission?
+    --> B(Account::ApplicationPolicy#grant_signin_permission?)
+
+    A --"UserUpdate.new(current_user, { supported_permission_ids: }, current_user, user_ip_address).call"
```

Comment:
It looks like the second argument is `params` (same for the destroy action and users equivalents), but are you just writing `{ supported_permission_ids: }` in its place to give a clearer flavour of what it is?

Reply:
Yeah I was writing out what params actually consist of here to make this a bit more useful